roofit/xroofit/src/xRooFit.cxx

```diff
@@ -10,9 +10,15 @@
  * listed in LICENSE (http://roofit.sourceforge.net/license.txt)
  */
 
+#include "RVersion.h"
+
+#if ROOT_VERSION_CODE < ROOT_VERSION(6, 27, 00)
 #define protected public
+#endif
 #include "RooFitResult.h"
+#if ROOT_VERSION_CODE < ROOT_VERSION(6, 27, 00)
```

Comment:
```suggestion
#ifdef protected
```
maybe that's more portable because one doesn't have to sync the conditions

Reply:
Thanks I'll update this in a future push, as I just squased

Reply:
actually did just now since had to push again to resolve compile warnings